### ISSUE TITLE
Embed links to last minutes of the group (?)

### ISSUE THREAD

**Issue by tripu** [w3c/Unitas#35]

**Comment by chaals** [comment]
It would be good to have a minutes tracker for each group, so people can hunt through them. Ideally, this would also enable collecting the topics from each set of minutes (at least the ones formatted by scribe.pl) onto a page so people can do a plain-text search for the discussion they are trying to find, from "some time a few years ago"…

I think this one is important, but probably not urgent (but watch the reaction counters). So it might be a task that should be broken into steps.

**Comment by tripu** [comment]
See if it's feasible to leverage @gosko's work on a client-side JS that digests a group's last minutes, extracts topics and renders an index with hyperlinks.

**Comment by tripu** [comment]
Open questions:

* How to handle authorisation for non-public meetings?
* How to map each WG's ID to their &ldquo;canonical&rdquo; IRC channel (which gives you the the pattern for the minutes URIs)?

**Comment by tripu** [comment]
(This is not just about "last meetings", but about "last topics. Even better!)

**Comment by chaals** [comment]
open questions...

* What about groups that have more than one channel / meeting? (I believe this applies to WebPlat, AG, payments, and perhaps others).

I don't think this should directly map by WG ID.

**Comment by tripu** [comment]
Right.
We were talking about extending our DB schema to map *group &harr; channel*.
Perhaps that should be *group &harr; {channels}*, then.

**Comment by chaals** [comment]
> Perhaps that should be _group ↔ {channels}_, then.

Yup. Since the various channels generally correspond to some defined sub-part of the group, it would be handy to be able to note that in the data.

**Comment by tripu** [comment]
> *&ldquo;How to map each WG's ID to their “canonical” IRC channel (which gives you the the pattern for the minutes URIs)?&rdquo;*

Confirming that the API does *not* return, among all fields, &ldquo;IRC channel(s)&rdquo; for a given group atm. See eg [`https://labs.w3.org/unitas/?g=83482#services`](https://labs.w3.org/unitas/?g=83482#services), [`https://labs.w3.org/unitas/?g=76043#services`](https://labs.w3.org/unitas/?g=76043#services), [`https://labs.w3.org/unitas/?g=35422#services`](https://labs.w3.org/unitas/?g=35422#services).
